api/models/node.go

```diff
@@ -35,6 +35,53 @@ func (i IndexList) Trace(span opentracing.Span) {
 func (i IndexList) TraceDebug(span opentracing.Span) {
 }
 
+type IndexTagFindSeries struct {
+	OrgId       int      `json:"orgId" binding:"Required"`
+	Expressions []string `json:"expressions"`
```

Comment:
I noticed in `models.GraphiteTagFindSeries{}` this filed is called `Expr`. let's name this field the same way?

Reply:
done :+1: